<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Motor</class>
 <widget class="QWidget" name="Motor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>144</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Motor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Motor Name</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QDial" name="motorDial">
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="notchTarget">
         <double>15.000000000000000</double>
        </property>
        <property name="notchesVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
